src/rules/orderedImportsRule.ts

```diff
@@ -109,6 +109,19 @@ function findUnsortedPair(xs: ts.Node[], transform: (x: string) => string): [ts.
     return null;
 }
 
+function sortByKey<T>(xs: T[], getSortKeyCallback: (x: T) => string): T[] {
+    return xs.sort((a, b) => {
+        const transformedA = getSortKeyCallback(a);
+        const transformedB = getSortKeyCallback(b);
+        if (transformedA > transformedB) {
+            return 1;
+        } else if (transformedA < transformedB) {
+            return -1;
+        }
+        return 0;
```

Comment:
Yeah, I used localeCompare, but it ignores casing, so it changes the behavior from what it was before